domains/client/domain-operator/src/tests.rs

```diff
@@ -34,7 +33,7 @@ use sp_domains::{
     InvalidBundleType, Transfers,
 };
 use sp_domains_fraud_proof::fraud_proof::{
-    ApplyExtrinsicMismatch, ExecutionPhase, FinalizeBlockMismatch, FraudProof,
+    ApplyExtrinsicMismatch, ExecutionPhase, FinalizeBlockMismatch, FraudProofVariant,
```

Comment:
Its already an enum with bunch of variants. Feels redundant to call it FraudProofVariant

Reply:
I agree, naming is hard as always :) WDYT `FraudProofType`?

Reply:
`FraudProof` is already used by
```rust
pub struct FraudProof<Number, Hash, DomainHeader: HeaderT, MmrHash> {
    ...
    pub proof: FraudProofVariant<Number, Hash, DomainHeader>,
}
```